### DISCUSSION TITLE
How to override border colour of Select component?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#4968] [Q&A]
Unlike the `Input` component, it seems the `Select` component doesn't have a theme var to override its border.

It seems (from inspecting the `.n-base-selection__border` element) it reads from the general `Common > borderColor` var.

![image](https://github.com/tusen-ai/naive-ui/assets/30174698/bbfc2cb8-676f-479e-9608-0597dd9a2fee)

But how to control `Select` border without resorting to common styles?

**Comment by jahnli** [comment]
套一层 config provider

**Reply by mitya33** [reply]
Could you elaborate? Do you mean I should set the `borderColor` variable under `Common` in the theme?

**Comment by jahnli** [comment]
https://www.naiveui.com/zh-CN/os-theme/docs/customize-theme#%E8%B0%83%E6%95%B4%E7%BB%84%E4%BB%B6%E4%B8%BB%E9%A2%98%E5%8F%98%E9%87%8F
